src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
create type for shape and color like this `type Shape = 'triangle' | 'circle' | 'rectangle';`